Keyboards/KeyboardsBase/KeyboardKeys.swift

```diff
@@ -291,25 +283,46 @@ class KeyboardKey: UIButton {
     } else if DeviceType.isPad {
       adjustPadKeyWidth()
     }
+  }
 
+  /// Adjusts the style of the button based on different states
```

Comment:
Minor nit that I'm noting here is that we'd like this to be a complete sentence with a period :) I can get it though!